catchup/service_test.go

```diff
@@ -421,98 +421,96 @@ func TestServiceFetchBlocksMalformed(t *testing.T) {
 	//require.True(t, s.fetcherFactory.(*MockedFetcherFactory).fetcher.client.closed)
 }
 
-func TestOnSwitchToUnSupportedProtocol(t *testing.T) {
```

Comment:
No changes in TestOnSwitchToUnSupportedProtocol except for dividing it into TestOnSwitchToUnSupportedProtocol1, TestOnSwitchToUnSupportedProtocol2, TestOnSwitchToUnSupportedProtocol3 and TestOnSwitchToUnSupportedProtocol4.